### PR TITLE
Add release notes for Decision Reviews v2 PDF downloads

### PR DESCRIPTION
**Pull request by caseywilliams**
These are release notes for three new endpoints in the Decision Reviews API v2. I'll add a link to our developer docs in the dev environment here ~~once they are available (this should be within the next few days).~~ Here are the docs in dev: https://dev-developer.va.gov/explore/api/decision-reviews/docs

### CODE REVIEW

File: content/appeals/decision-reviews/release-notes/2023-09-04.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+New endpoints have been added to allow downloads of submitted PDFs. When you submit a Higher-Level Review, or a Notice of Disagreement, a Supplemental Claim, you can now access a watermarked copy of the PDF as submitted to the VA with the following caveats:
+
+1. A PDF download will become available only after the appeal has progressed to the 'submitted' state.
+2. The PDF will stop being available one week after the appeal has progressed to the 'completed' state (when the veteran's personally identifying information is purged from our severs).
```
Comment (joeabraun) [inline content/appeals/decision-reviews/release-notes/2023-09-04.md:4]:
```suggestion
2. The PDF will stop being available one week after the appeal has progressed to the 'completed' state. This is when the Veteran's personally identifying information is purged from our severs.
```

File: content/appeals/decision-reviews/release-notes/2023-09-04.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+3. PDFs are only available for appeals created with an associated veteran ICN, which is provided in the `X-VA-ICN` header when the appeal is first created. If the appeal was not created with an `X-VA-ICN` header, a PDF will never become available.
```
Comment (joeabraun) [inline content/appeals/decision-reviews/release-notes/2023-09-04.md:5]:
```suggestion
3. PDFs are only available for appeals created with an associated Veteran ICN, which is provided in the `X-VA-ICN` header when the appeal is first created. If the appeal was not created with an `X-VA-ICN` header, a PDF will never become available.
```

File: content/appeals/decision-reviews/release-notes/2023-09-04.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+The PDF download endpoints are:
+* `/services/appeals/v2/decision_reviews/higher_level_reviews/ID/download`
```
Comment (joeabraun) [inline content/appeals/decision-reviews/release-notes/2023-09-04.md:8]: Will the asterisks render as bullet points in prod?

**Review by joeabraun — CHANGES_REQUESTED** [review]
Requested some changes from UX.

File: content/appeals/decision-reviews/release-notes/2023-09-04.md (hunk 3 above)
Comment (caseywilliams) [reply content/appeals/decision-reviews/release-notes/2023-09-04.md:8]: If prod uses standard-compliant markdown, then yes, but just to be safe, I'll copy what I'm seeing in a few other release notes from before, where dashes were used instead

File: content/appeals/decision-reviews/release-notes/2023-09-04.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+New endpoints have been added to allow downloads of submitted PDFs. When you submit a Higher-Level Review, a Notice of Disagreement, or a Supplemental Claim, you can now access a watermarked copy of the PDF as submitted to the VA with the following caveats:
+
+1. A PDF download will become available only after the appeal has progressed to the 'submitted' state.
+2. The PDF will stop being available one week after the appeal has progressed to the 'completed' state. This is when the Veteran's personally identifying information is purged from our severs.
```
Comment (joeabraun) [inline content/appeals/decision-reviews/release-notes/2023-09-04.md:4]:
```suggestion
2. The PDF will stop being available one week after the appeal has progressed to the 'completed' state. This is when the Veteran's personally identifying information is purged from our servers.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]
This

**Review by joeabraun — APPROVED** [review]
Looks good to me! UX approves.

**Pull request merged**